examples/README.md

```diff
@@ -0,0 +1,113 @@
+# Examples
+
+this directory contains examples for a few features of the autograder
+
+## Assessments
+
+The Hello folder and Hello.tar contain various examples of different styles of Assessments
+
+
+## Importing Assessments
+
+whether you are importing one or more assessments the procedure is the same
+
+multiple assessment import starter.tar is the starter file for importing one or more assessments
+
+
+```bash
+multiple assessment import starter.tar
+├── assessment1
+│   ├── assessment1.yml
+│   ├── autograde.tar
+│   └── autograde-Makefile
+└── assessment2
+    └── assessment1.yml
+
+```
+
+every folder in the tar represents a separate assessment
+
+assessment1 is an example of an assessment with an autograding component
+
+properties.yml
+```YAML
+---
+general:
```

Comment:
I think there should be a configuration version number here that is bumped any time the `properties.yaml` or tarball format changes.

Reply:
there is its
book_keeping:
  interface_number: 1000
  
  i just made it 1000 to look more official than just a 1